schema/deploy/shipping/age-bin-v2.sql

```diff
@@ -0,0 +1,65 @@
+-- Deploy seattleflu/schema:shipping/age-bin-v2 to pg
+-- requires: types/intervalrange
+
+begin;
+
+create table shipping.age_bin_fine_v2 (
+    range intervalrange primary key,
+
+    constraint age_only_in_one_fine_range_v2
+        exclude using gist (range with &&)
+);
+
+create index age_bin_fine_range_idx_v2 on shipping.age_bin_fine_v2 using gist(range);
+
+comment on table shipping.age_bin_fine_v2 is 
+    'Version 2 of fine-grained age bins that uses intervalrange type';
+comment on column shipping.age_bin_fine_v2.range is
+    'The range of each age bin defined as intervalrange';
+
+insert into shipping.age_bin_fine_v2 values
+    ('[0 mon,2 mons)'::intervalrange),
+    ('[2 mons,7 mons)'::intervalrange),
+    ('[7 mons,2 years)'::intervalrange),
+    ('[2 years,5 years)'::intervalrange),
```

Comment:
I'm not sure if these first four bins are quite right, based on my interpretation of [this Slack message](https://seattle-flu-study.slack.com/archives/CE9B78CS0/p1556051395043700) and [this other message](https://seattle-flu-study.slack.com/archives/CE9B78CS0/p1555982567033000). I interpreted it as:

```
[0 months, 1 month)   -- first month of life
[1 month, 6 months)   -- before 6 months
[6 months, 1 year)    -- before 1 year
[1 year, 5 years)     -- before 5 years
```

Thoughts? I know I found those two messages I linked to a bit confusing because they leave gaps between suggested ranges and/or don't specify inclusive/exclusive bounds, so maybe we should just ask for clarification.